src/test/testconfig.sh.example

```diff
@@ -64,8 +64,10 @@
 # to all of the following resource files (containing physical addresses)
 # of NVDIMM devices (only root can read them by default):
 #
-# /sys/bus/nd/devices/ndbus*/region*/resource
-# /sys/bus/nd/devices/ndbus*/region*/dax*/resource
+# /sys/bus/nd/devices/ndbus*/region*/<resource>
+# /sys/bus/nd/devices/ndbus*/region*/dax*/<resource>
+# or
+# /sys/devices/platform/e820_pmem/ndbus*/region*/dax*/<resource>
```

Comment:
isn't this just a symlink to `/sys/bus/nd/devices/ndbus*/region*/dax*/<resource>` ?